core/tauri/scripts/process-ipc-message-fn.js

```diff
@@ -0,0 +1,29 @@
+// Copyright 2019-2023 Tauri Programme within The Commons Conservancy
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-License-Identifier: MIT
+
+(function (message) {
```

Comment:
probably want to open with a semicolon here
```suggestion
;(function (message) {
```